starfish/experiment/experiment.py

```diff
@@ -41,10 +46,8 @@ class FieldOfView:
     PRIMARY_IMAGES = 'primary'
 
     def __init__(
-            self,
-            name: str,
-            images: Optional[MutableMapping[str, ImageStack]]=None,
-            image_tilesets: Optional[MutableMapping[str, TileSet]]=None,
+            self, name: str,
+            image_tilesets: Optional[MutableMapping[str, TileSet]] = None
```

Comment:
Since we no longer accept ImageStacks _or_ TileSets, the `image_tilesets` parameter should be required.

```suggestion
            image_tilesets: MutableMapping[str, TileSet],
```